app/models/Post.js

```diff
@@ -33,11 +33,10 @@ const PostSchema = new Schema({
         default: 0
       },
       users: {
-        user: {
+        user: [{
```

Comment:
why are you doing this? as in users then inside that user array. cant we just use users and have array of objects inside that?

Reply:
I did not want to disturb the predefined schema. I was also thinking that having just users and have an array of objects inside that would be better. Will do it that way.